docs/0-zarf-overview.md

```diff
@@ -159,39 +159,77 @@ In the more complex use case, your package consists of updates for many apps / s
 
 ## Quick Start
 
-:::info
+1.  Select your systems OS below
+2.  Ensure you have the pre-requisite applications running
+3.  Enter the commands into your terminal
+
+<Tabs>
+<TabItem value="macOS" label="macOS" default>
+  
+ :::info
```

Comment:
The `:::info` blocks don't seem to work as intended when inside of the above `<Tabs> / <TabItem>` blocks. This is what it looks like rendered out.

<img width="861" alt="Screen Shot 2022-10-10 at 12 27 34 PM" src="https://user-images.githubusercontent.com/18587823/194912815-1fb3b619-4b92-4900-8793-75959dcb8cb0.png">

Reply:
@YrrepNoj Yes I need some help with that. I remember that the same thing was happening when I put a code block in the tab section. @mike-winberry was able to figure out why (something with markdown?)